.github/CODEOWNERS

```diff
@@ -27,14 +27,19 @@ yarn.lock @ecraig12345 @dzearing @xugao
 *.yml @ecraig12345
 tsconfig.json @ecraig12345
 lerna.json @ecraig12345
-LICENSE @ecraig12345
-.git* @ecraig12345
 .codesandbox @ecraig12345
+.devops @ecraig12345
 .eslintrc.* @ecraig12345
+.githooks @ecraig12345
```

Comment:
I think this was also intended to cover .gitattributes and .gitignore. Not sure how important either of those actually are. .gitattributes is almost never modified but could cause some pretty dramatic issues if changed.

Reply:
Oh, my bad I was just looking at folders. I knew there were more .git* folders/files just didn't see them. I'll fix it.

Reply:
@ecraig12345 - Fixed!